understanding/20/contrast-minimum.html

```diff
@@ -13,15 +13,14 @@ <h1>Understanding Contrast (Minimum)</h1>
       <h2>Intent of Contrast (Minimum)</h2>
       
       
-	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with moderately low vision.
+	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by people with limited vision, including color vision deficiencies.
```

Comment:
My concern is that that describes a condition, but we're actually talking about people. And there was a prior inference in the language that the WG understood this didn't solve thing for everyone.

We can solve the latter by the addition of "more" can give us both...
```suggestion
	<p>The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be read by more people with limited vision, including color vision deficiencies.
```

Reply:
I prefer the original.  This is in the context of intent, but I agree that the intent should not over state the efficacy of the SC.  How about:

> The intent of this Success Criterion is to provide enough contrast between text and its background so that it can be more easily by some people with limited vision, including some people with color vision deficiencies.

Reply:
I like both new suggestions.